department-of-veterans-affairs/abd-vro
# Cleanup svc-assessor-7101v2 docker image

## What was the problem?

Before, the code for MAS integration was in its own docker image. Queue name `health-assessment.7101v2` was confusing. 

## How does this fix it?[^1]

Now the updated logic for assessing sufficiency of health data is a part of the svc-assessor-dc7101. Still a queue is set up, `health-sufficiency-assessment.7101`, the same as it was before so it should not affect up-stream procedures. This way hypertension code and logic is more condensed and we have less docker images and duplicate code to deal with. Unit testing has also been overhauled for the new MAS integration logic. 

Additionally the new queue is set up to return evidence for the new PDF. (BP readings in the past two year, conditions are included)

## How to test this PR
- Step 1
- Try MAS integration workflow and make sure it connects to the queue
- Step 2[^secrel]


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
